### DISCUSSION TITLE
Frient = Develco ?

### DISCUSSION THREAD

**Discussion by HerrieMan72** [Koenkk/zigbee2mqtt#7578] [Devices]
Hi All,
i'm looking to buy new smoke/fire sensors for my home and the Develco ones listed https://www.zigbee2mqtt.io/devices/SMSZB-120.html and https://www.zigbee2mqtt.io/devices/HESZB-120.html look very nice but I can't find any webshop where to buy these here in the Netherlands.
These devices look very similar to what is offered by Frient https://frient.com/products/intelligent-smoke-alarm/ and https://frient.com/products/intelligent-heat-alarm/ and their sensors are available via various webshops here in the Netherlands. Can someone please confirm if these are the same sensors and that also the Frient devices are supported by zigbee2mqtt ?
Thanks,
Herman

**Comment by HerrieMan72** [comment]
Hi,
I bought four of the Frient devices, taking the risk that these would be supported like the Develco devices, but the results are mixed.
The two SMSZB-120 smoke detectors are correctly recognised and show as supported in the list of devices:
![image](https://user-images.githubusercontent.com/34721776/119558207-682b8900-bda1-11eb-82cb-0b0cf3663935.png)

But the two HESZB-120 fire detectors are showing up as unsupported:
![image](https://user-images.githubusercontent.com/34721776/119558399-a1fc8f80-bda1-11eb-8c5a-7d21c2cc558e.png)

I have been looking at the links on how to support new devices, but to be honest I don't really know how to approach this or what I need to do to get these firedetectors also recognised, they seem to be equal to the ones listed here: https://www.zigbee2mqtt.io/devices/HESZB-120.html

Any help would be greatly appreciated.

**Reply by HerrieMan72** [reply]
BTW i'm running zigbee2mqtt version 1.18.1 as an addon in my supervised home-assistant.
I've been trying to get the latest https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/develco.js file running on my system as an external-converter, but I then get all kinds of missing dependencies or files.

**Reply by Koenkk** [reply]
Device should be supported in 1.18.3 (but you probably need to switch to the new repo to get the update: https://github.com/zigbee2mqtt/hassio-zigbee2mqtt)

**Reply by HerrieMan72** [reply]
Hi Koen,
thanks for your reply, switching the repo indeed did help, apperantly I was using an other repo that was stuck on the 1.18.1 release or so.
Yesterday I tried fixing the external-converter and ended up with cloning the full zigbee-herdsman-converters repo into the working folder of zigbee2mqtt, that fixed the missing dependencies from the develco.js file, i'm not sure however if this is the way it's supposed to be done. I'm not sure if the needed files are maybe already somewhere else on my system but that I simply couldn't find them.

**Reply by HerrieMan72** [reply]
I have yet another question about these sensors.
I would like to pair or group these sensors, so that when one sensor detects smoke/fire all sensors start to give an alarm. When reading the technical manual: https://www.develcoproducts.com/media/1742/smszb-120-technical-manual-smoke-alarm.pdf it seems these devices have some sort of zone configuration where they can share status or something.
I do not fully understand all the details of zigbee, i've tried pairing them via the ez-mode as described in the manual, but that doesn't seem to work (or I'm doing something wrong). Also i've tried to group them via zigbee2mqtt, but that also doesn't work.
Does anyone know if it is possible to join them ?

**Comment by sjorge** [comment]
Oh nice, I was looking for some of the Develco ones but couldn't find any in belgium either, my Heimann smoke detector gets triggerd if I am cooking a lot so heat based one would be amazing.

Might pick one up next month
